snuba/datasets/errors_replacer.py

```diff
@@ -393,20 +393,39 @@ def process_tombstone_events(
         # not.
         return None
 
-    # XXX: We need to construct a query that works on both event_id columns,
-    # either represented as UUID or as hyphenless FixedString. That's why we
-    # use replaceAll(toString()).
-    where = """\
-        PREWHERE replaceAll(toString(event_id), '-', '') IN (%(event_ids)s)
+    from_ts = message.get("from_timestamp")
+    to_ts = message.get("to_timestamp")
+    if from_ts is not None and to_ts is not None:
```

Comment:
Not sure how we expect Sentry to use these but would it be better to handle to and from independently? As far as I can tell, we could still benefit from either being present (it would be better than having neither)

Reply:
Made the two independent